.prettierrc

```diff
@@ -0,0 +1,10 @@
+useTabs: true
+tabWidth: 4
+printWidth: 80
+singleQuote: true
+trailingComma: es5
+bracketSpacing: true
+parenSpacing: true
+jsxBracketSameLine: false
+semi: true
+arrowParens: always
```

Comment:
Thanks for pointing this out! I've fixed the config file now 🙂. Don't know why it wasn't right in the first place 🤔.

>How about adding .prettierignore

I don't think this is necessary. Right now all the formatting commands only look for JS and CSS files. There's also a command to selectively format only provided files.

The pre-commit hook is also only run on files you actually changed, so I don't think this will introduce a significant enough delay in commits that it's worth it to try to speed it up?

Reply:
> I don't think this is necessary. Right now all the formatting commands only look for JS and CSS files under assets and client. There's also a command to selectively format only provided files.

This should improve scanning speed for js and css files. But I agree we can add it later if necessary